subgraph/subgraph.yaml

```diff
@@ -0,0 +1,156 @@
+specVersion: 0.0.2
+description: clr.fund sandbox
+repository: https://github.com/daodesigner/clrfund-deployer
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: FundingRoundFactory
+    network: xdai
+    source:
+      address: '0x549F91A93c94358C5f5380D7ABF23E1340CfF2db'
+      abi: FundingRoundFactory
+      startBlock: 15217676
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - FundingRoundFactory
+        - RecipientRegistry
+        - ContributorRegistry
+        - FundingRound
+      abis:
+        - name: FundingRoundFactory
+          file: ./abis/FundingRoundFactory.json
+        - name: FundingRound
+          file: ./abis/FundingRound.json
+        - name: MACIFactory
+          file: ./abis/MACIFactory.json
+        - name: OptimisticRecipientRegistry
+          file: ./abis/OptimisticRecipientRegistry.json
+        - name: BrightIdUserRegistry
```

Comment:
So to be clear, this subgraph will only work when using OptimisticRecipientRegistry & BrightIdUserRegistry - is that correct?

Reply:
Wondering what challenges that creates if the only way we can now run the dev environment is with the subgraph 🤔

Reply:
There is a fall back but you wouldn't get things like sponsored or verified which would break the front end. Will open a ticket to add the other registry types down the line.